src/ThreeEditor/components/Sidebar/propeteries/category/PropertiesCategory.tsx

```diff
@@ -0,0 +1,25 @@
+import { Accordion, AccordionSummary, Typography, AccordionDetails, Grid } from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+
+export function PropertiesCategory(props: {
+	category: string;
+	children: React.ReactNode;
+	visible?: boolean;
+}) {
+	const { category, children, visible = true } = props;
+
+	return (
+		<Accordion key={category} sx={{ display: visible ? '' : 'none' }}>
+			<AccordionSummary expandIcon={<ExpandMoreIcon />}>
+				<Typography>{category}</Typography>
+			</AccordionSummary>
+			<AccordionDetails>
+				{visible && (
```

Comment:
This check seems to be redundant because we already check it outside of the component. This should work without it or if you can make it work with generic check you can remove specific ones on the outside.

Reply:
True